pdbx/containers.py

```diff
@@ -176,54 +232,75 @@ class DefinitionContainer(ContainerBase):
     """Container for definitions."""
 
     def __init__(self, name):
+        """Initialize container with name.
+
+        :param str name:  name of container
+        """
         super(DefinitionContainer, self).__init__(name)
         self.set_type("definition")
 
-    def is_category(self):
-        """Determine if container contains category objects."""
+    def is_category(self) -> bool:
+        """Determine if container contains category objects.
+
+        :returns:  indicator of whether category objects are in container
+        """
         if self.exists("category"):
             return True
         return False
 
-    def is_attribute(self):
-        """Determine if container contains item objects."""
+    def is_attribute(self) -> bool:
+        """Determine if container contains item objects.
+
+        :returns:  indicator of whether item objects are in container
+        """
         if self.exists("item"):
             return True
         return False
 
-    def print_it(self, fh=stdout, type_="brief"):
-        """Print information about container to file object."""
-        fh.write(
+    def print_it(self, file_=stdout, type_="brief"):
```

Comment:
Renaming of arguments is a backwards incompatible change and would require a new major version.

Reply:
Ugh.  It bugged me because the original code changed the arguments from the base class... but I'll reverse the change.

Reply:
Fixed in https://github.com/Electrostatics/mmcif_pdbx/pull/47/commits/2ce6490f21c58b9181db8624ac04f54daec2d27d